src/main/java/ecdar/controllers/EcdarController.java

```diff
@@ -522,10 +523,21 @@ public static void setActiveCanvasPresentation(CanvasPresentation newActiveCanva
         activeCanvasPresentation.get().setOpacity(0.75);
         newActiveCanvasPresentation.setOpacity(1);
         activeCanvasPresentation.set(newActiveCanvasPresentation);
+        updateZoomShortcutBindings(newActiveCanvasPresentation.getController().zoomHelper);
+    }
+
+    /**
+     * Binds the zoom shortcuts to the provided zoom helper (consequently unbinds the previously bound zoom helper)
+     */
+    private static void updateZoomShortcutBindings(ZoomHelper zoomHelper) {
```

Comment:
"update" could imply that we could change the ```KeyCode``` to another. Maybe ````registerZoomShortcutBindings``` would be more clear?

Reply:
Hmm, as written in the comment, this unbinds the currently bound `ZoomHelper` and practically replaces it with the new one. There might be a better word than update, but register implies that the currently bound methods are preserved IMO. Maybe _rebind_?

Reply:
Rebind makes sense. :+1: